### ISSUE TITLE
Error in SV generation for a task with no retval or param

### ISSUE THREAD

**Issue by djoffe** [fvutils/pyhdl-if#1]
Hello,

I appreciate this tool; it has been saving me time with my Python-DPI interactions!

A task that lacks a return value or parameters is generated without a semicolon, leading to invalid SystemVerilog code.

```py
@hif.api
class FooDPI(object):
    @hif.imptask
    async def say_foo(self):
        pass
```

```sv
    virtual task invokeTask(
        output pyhdl_if::PyObject retval,
        input string method,
        input pyhdl_if::PyObject args);

        retval = pyhdl_if::None;

        case (method)
            "say_foo": begin
                say_foo            end
            default: begin
                $display("Fatal: unsupported method call %0s", method);
            end
        endcase
    endtask
```

**Comment by mulianov** [comment]
@djoffe Line 418 fix this issue by adding proper (). Please merge it asap

**Comment by cmarqu** [comment]
Closed by #2.
